test_basic.c

```diff
@@ -67,11 +68,12 @@ static void basic_draw(GtkWidget * widget)
 	gint w, h;
 	gint i, b, d;
 	gint maxwidth, maxheight;
-	gint widths[6];
+        gint widths[7];
```

Comment:
Formatting is currently TABs throughout the project. Could you try to keep the formatting consistent?

Reply:
Ohhh sorry. Yes it is no problem. Qr Creator has filled the tab with space at me! I have this changed in the settings. :)